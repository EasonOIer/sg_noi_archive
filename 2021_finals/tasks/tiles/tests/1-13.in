7 6
.....xx
.x.x...
x..x.xx
2 1 5
1 2 6
2 1 7
2 4 6
2 2 5
2 1 4
