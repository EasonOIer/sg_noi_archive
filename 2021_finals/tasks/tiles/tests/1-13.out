132
132
5
49
30
